xbmc/windowing/X11/GLContextEGL.cpp

```diff
@@ -17,17 +17,22 @@
  *  <http://www.gnu.org/licenses/>.
  *
  */
+#include "system_gl.h"
```

Comment:
We need this header for the definition of HAVE_X11 and HAS_EGL, as well as for glFinish. Alternatively I can use system.h for the former, but the latter requires both GLES2/GL(due to https://github.com/FernetMenta/xbmc/blob/master/xbmc/windowing/X11/GLContextEGL.cpp#L160 ) headers. Since they are conflicting with each other I have to replicate https://github.com/lkosmid/xbmc/blob/EGL_refactoring/xbmc/system_gl.h#L25 -52, which in essence inlines "system_gl.h".